Set DYLD_LIBRARY_PATH when running tests so that the just-built libraries
can be found and used.
--- cmake/LibKMLHelper.cmake.orig	2015-12-21 11:23:05.000000000 -0600
+++ cmake/LibKMLHelper.cmake	2019-04-28 11:50:32.000000000 -0500
@@ -48,6 +48,7 @@
   add_dependencies(test_${TEST_NAME} ${TEST_DEPENDS})
   target_link_libraries(test_${TEST_NAME} ${TEST_DEPENDS} ${TEST_LINKS} ${GTEST_LIBRARY})
   add_test(${TEST_GROUP}_${TEST_NAME} ${CMAKE_BINARY_DIR}/bin/test_${TEST_NAME})
+  set_tests_properties(${TEST_GROUP}_${TEST_NAME} PROPERTIES ENVIRONMENT DYLD_LIBRARY_PATH=${CMAKE_BINARY_DIR}/lib:$ENV{DYLD_LIBRARY_PATH})
 endfunction(build_test)
 
 function(build_example)
